db/schema.rb

```diff
@@ -213,9 +173,9 @@
     t.string   "required_ruby_version"
     t.string   "sha256"
     t.hstore   "metadata",                  default: {},   null: false
+    t.datetime "yanked_at"
```

Comment:
It's been doing this on my local copy too, guessing it's just committed in the wrong order?